impl/src/main/java/org/jboss/cdi/tck/AbstractTest.java

```diff
@@ -46,6 +47,7 @@
  */
 public abstract class AbstractTest extends Arquillian {
 
+    // TODO this will have to be eventually changes as Lite implementations do not have BeanManager, only BeanContainer
```

Comment:
I am aware of this TODO and left it here deliberately.
We will have to alter this abstract class somehow so it works for Lite and Full implementations (as in Lite you cannot inject `BeanManager`).

Reply:
One option is we could tighten up the spec a bit. Currently, we say that even injecting (or otherwise obtaining) a `BeanManager` is non-portable -- we could perhaps remove that. We'd keep the provision that calling `BeanManager` methods (those that are not inherited from `BeanContainer`) is non-portable, obviously.

Reply:
Yeah, that would work fine. I didn't want to solve it as part of this commit though so I only left a TODO note.